content/en/docs/managing-jx/common-tasks/tls.md

```diff
@@ -6,4 +6,21 @@ date: 2019-08-28
 weight: 190
 ---
 
-Please see the [boot ingress documentation](/docs/getting-started/setup/boot/#ingress)
\ No newline at end of file
+As documented in the [boot ingress documentation](/docs/getting-started/setup/boot/#ingress), you can configure a global default certificate for all Jenkins X ingresses.
+This requires using [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/) for a wildcard certificate, or manually adding and updating your certificate.
```

Comment:
Some suggestions and typos:

> This requires using [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/) for a wildcard certificate, or manually adding and updating your certificate.

This requires using cert-manager's [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/) challenge provider for wildcard certificates, or manually adding and updating your certificate.

> In case this is not practicle and you want to generate one certificate per ingress using [http01](https://cert-manager.io/docs/configuration/acme/http01/), you can specify the `secreName`s to be used for each ingress in the `values.yaml` as described below

practicle == practical 
secreName == secretName

Reply:
@hferentschik here you go :), thanks for spotting the typos, my bad